api/scripts/prod/create-prolife-collection-campaigns-for-sco.js

```diff
@@ -0,0 +1,92 @@
+// Usage: node create-profile-collection-campaigns-for-sco.js path/file.csv <creatorId>
+// To use on file with columns |name, customLandingPageText, organizationsId|, those headers included
+const bluebird = require('bluebird');
+const { knex } = require('../../db/knex-database-connection');
+const Campaign = require('../../lib/domain/models/Campaign');
+const campaignCodeGenerator = require('../../lib/domain/services/campaigns/campaign-code-generator');
+const campaignValidator = require('../../lib/domain/validators/campaign-validator');
+const campaignRepository = require('../../lib/infrastructure/repositories/campaign-repository');
+const { parseCsvWithHeader } = require('../helpers/csvHelpers');
+
+function checkData(campaignData) {
+  return campaignData.map(({ name, organizationId, customLandingPageText }, index) => {
+    if (!organizationId) {
+      throw new Error(`Ligne ${index + 1}: Une organizationId est obligatoire pour la campagne de collecte de profile.`);
+    }
+
+    if (!name) {
+      throw new Error(`Ligne ${index + 1}: Un nom de campagne est obligatoire pour la campagne de collecte de profile.`);
+    }
+
+    return { name, organizationId, customLandingPageText };
+  });
+}
+
+async function prepareCampaigns(campaignsData, creatorId) {
+  const campaigns = await bluebird.mapSeries(campaignsData, async (campaignData) => {
+
+    const campaign = {
+      creatorId,
+      type: Campaign.types.PROFILES_COLLECTION,
+
+      name: campaignData.name,
+      customLandingPageText: campaignData.customLandingPageText || '',
+      organizationId: campaignData.organizationId,
+    };
+
+    campaignValidator.validate(campaign);
+    campaign.code = await campaignCodeGenerator.generate(campaignRepository);
+
+    if (require.main === module) process.stdout.write(`Collecte de Profile ${ campaign.name } pour l'organisation ${ campaign.organizationId } ===> ✔\n`);
+    return campaign;
+  });
```

Comment:
Je serai presque tentée de convertir ce mapSeries en map histoire de pouvoir caler une petite concurrency et aller un peu plus vite snas que cela n'impacte trop la BDD ->
```suggestion
  const campaigns = await bluebird.map(campaignsData, async (campaignData) => {

    const campaign = {
      creatorId,
      type: Campaign.types.PROFILES_COLLECTION,

      name: campaignData.name,
      customLandingPageText: campaignData.customLandingPageText || '',
      organizationId: campaignData.organizationId,
    };

    campaignValidator.validate(campaign);
    campaign.code = await campaignCodeGenerator.generate(campaignRepository);

    if (require.main === module) process.stdout.write(`Collecte de Profile ${ campaign.name } pour l'organisation ${ campaign.organizationId } ===> ✔\n`);
    return campaign;
  }, { concurrency: 5 });
```